src/ensembl/src/shared/types/thoas/metadata.ts

```diff
@@ -22,11 +22,19 @@ export type ValueSetMetadata = {
 
 type CanonicalMetadata = Omit<ValueSetMetadata, 'value'> & { value: boolean };
 
+type NCBITranscriptMetadata = {
+  id: string;
+  url: string;
+};
+type MANEMetadata = ValueSetMetadata & {
+  ncbi_transcript: NCBITranscriptMetadata | null;
```

Comment:
According to Thoas, ncbi_transcript is always present:

<img src="https://user-images.githubusercontent.com/6834224/126649019-c8de9281-b894-4f71-9f2b-0a9dcf0d2ed0.png" width="500" />

Don't know if it's correct or not, but that's what Thoas promises, and it will break if it can't find ncbi_transcript for transcript mane metadata

Reply:
ok we need to confirm this

Reply:
https://genomes-ebi.slack.com/archives/C0J0Z9F96/p1626967430036300